app/serializers/api/v1/rent_serializer.rb

```diff
@@ -1,7 +1,7 @@
 module Api
   module V1
     class RentSerializer < ActiveModel::Serializer
-      attributes :id, :init_date, :return_date, :book_id, :user_id
+      attributes :id, :init_date, :return_date, :book, :user
```

Comment:
Hum, this will expend the entire record from book and user? 
What happen for other endpoint you ever craft and use this serializer? Or there is not?